paddle/framework/CMakeLists.txt

```diff
@@ -5,3 +5,4 @@ nv_test(dim_test SRCS dim_test.cu DEPS ddim)
 cc_test(variable_test SRCS variable_test.cc)
 cc_test(scope_test SRCS scope_test.cc)
 cc_test(enforce_test SRCS enforce_test.cc)
+proto_library(op_proto SRCS op_proto.proto attr_type.proto)
```

Comment:
proto_library doesn't contain `DEPS` argument. Each protobuf file can compile to the independent library and doesn't depends on other files.